network-api/networkapi/wagtailpages/utils.py

```diff
@@ -469,3 +469,10 @@ def get_default_locale():
 def get_original_by_slug(Model, slug):
     (DEFAULT_LOCALE, DEFAULT_LOCALE_ID) = get_default_locale()
     return Model.objects.get(slug=slug, locale=DEFAULT_LOCALE_ID)
+
+
+def require_share_fields(form):
+    # Overrides a page's admin form in order to make promote tab's
+    # "search image" and "search description" fields required.
+    form.fields["search_description"].required = True
+    form.fields["search_image"].required = True
```

Comment:
What do you think about keeping this in the `forms.py`? I am not sure this can be reused anywhere else, this seems very specific. It is also only two lines and no logic but rather configuration. What do you think about inlining this instead of the function? I usually like this approach of extracting functionality into named functions, but this seems like an odd case somehow 🤔